### PR TITLE
BNCASB-2204: Initial changes to clean the project (removing group client and unused code)

### PR DESCRIPTION
**Pull request by rohandev**
Initial changes to clean the project (removing group client and unused code)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,42 +69,13 @@ The retry call supports the `delay` and `execute_inline` arguments in order to d
...
-
-For example:
-
-```bash
-python manage.py run_eb_sqs_worker --url http://127.0.0.1:80/worker/process --queue default
-```
-
-#### Executing Tasks without Elastic Beanstalk
-
 Another way of executing tasks is to use the Django command `process_queue`.
```
Comment (alexeyts) [inline README.md]: We should rephrase...

File: eb_sqs/worker/worker.py [hunk 2]
```diff
@@ -136,52 +126,4 @@ def _execute_task(self, worker_task):
             result = worker_task.execute()
             return result
         finally:
```
Comment (alexeyts) [inline eb_sqs/worker/worker.py]: `finally` clause can be removed

File: setup.py [hunk 3]
```diff
@@ -10,13 +10,12 @@
     package_dir={'eb_sqs': 'eb_sqs'},
```
Comment (alexeyts) [inline setup.py]: we still need to bump the version manually here, so we'll be able to consume the change

File: README.md (hunk 1 above)
Comment (rohandev) [reply README.md]: ok.Sure.

File: eb_sqs/worker/worker.py (hunk 2 above)
Comment (rohandev) [reply eb_sqs/worker/worker.py]: ok.

File: setup.py (hunk 3 above)
Comment (rohandev) [reply setup.py]: ok.

**Pull request merged**